app/models/user.rb

```diff
@@ -17,6 +17,7 @@ class User < ApplicationRecord
   validates :email, presence: true, uniqueness: {case_sensitive: false}
   validates :first_name, :last_name, :status_group, presence: true
   validates :password_set, inclusion: [true, false]
+  validates :openai_api_key, allow_blank: true, length: {maximum: 255}
```

Comment:
Where does this limit of 255 come from? Is it imposed by OpenAI or some assumption?

Reply:
I think the max length is 51. Was not sure if I should make it strict to that or not as things might change. 255 seemed reasonable